skel/share/defaults/transfermanagers.properties

```diff
@@ -85,18 +85,19 @@ transfermanagers.limits.transfer-time = 7200
 (one-of?true|false|${dcache.enable.space-reservation})transfermanagers.enable.space-reservation=${dcache.enable.space-reservation}
 
 # Database settings
+# Obsolete
 (one-of?true|false)transfermanagers.enable.db=false
-transfermanagers.db.host = ${dcache.db.host}
-transfermanagers.db.name = transfermanagers
-transfermanagers.db.user = ${dcache.db.user}
-transfermanagers.db.password = ${dcache.db.password}
-transfermanagers.db.password.file = ${dcache.db.password.file}
-transfermanagers.db.url=jdbc:postgresql://${transfermanagers.db.host}/${transfermanagers.db.name}?targetServerType=master
-(prefix)transfer-manager.db.hikari-properties = Hikari-specific properties
+(obsolete)transfermanagers.db.host = ${dcache.db.host}
```

Comment:
The value of obsolete properties should return a meaningful error message, like:

```
(obsolete)transfermanagers.db.host = DB functionality is removed
```

Reply:
I've removed it in my latest commit